drivers/mbox/Kconfig

```diff
@@ -9,20 +9,16 @@ menuconfig MBOX
 
 if MBOX
 
+# Include these first so that any properties (e.g. defaults) below can be
+# overridden (by defining symbols in multiple locations)
+source "drivers/mbox/Kconfig.nrfx"
```

Comment:
Minor comment (non-blocking): Is there an MBOX driver that needs to override `CONFIG_MBOX_INIT_PRIORITY`? I assume that this pattern was followed due to the way that the "root" MDIO drivers KConfig file appears to be written, but in my experience the individual driver KConfig files are typically sourced after the primary KConfig file has defined all symbols